### PR TITLE
Fix inaccurate Pulsar and Bookkeeper metrics

### PR DESCRIPTION
**Pull request by CodePrometheus**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
                     "name": "bookie_ledgers_count"
                   },
                   "expressions": [
-                    "meter_bookkeeper_bookie_ledgers_count"
+                    "latest(aggregate_labels(meter_bookkeeper_bookie_ledgers_count,sum))"
```
Comment (wankai123) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json:37]: if the weight type is `Line`, we should not use `latest`. Its result is not a `TIME_SERIES_VALUES`

**Comment by wankai123** [comment]
Please attach the screen-shoots when you update the UI templates

**Comment by wankai123** [comment]
Here is an illegal expression, please fix it as well, thanks.
https://github.com/apache/skywalking/blob/a38628fced0a6f678df188799b60c29c0507bc02/oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-broker.json#L85

**Comment by CodePrometheus** [comment]
broker-cluster
bookkeeper-cluster

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json (hunk 1 above)
Comment (CodePrometheus) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json]: Thanks for the reminder, fixed.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json [hunk 2]
```diff
@@ -214,7 +214,7 @@
                   "i": "15",
                   "type": "Widget",
                   "expressions": [
-                    "meter_pulsar_storage_read_rate.sum(['cluster','node'])"
+                    "meter_pulsar_storage_read_rate"
```
Comment (wankai123) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json:217]: Doesn't `pulsar-cluster` require `aggregate_labels ` to aggregate the nodes?

**Comment by wankai123** [comment]
On the `Pulsar-Broker` dashboard, why some of the metrics only have 1 datapoint value?
image

**Comment by wankai123** [comment]
suggest adding `label` in `metricConfig` for all of the dashboards, otherwise, the MQE expression would show on the weight directly.
image

**Comment by CodePrometheus** [comment]
It is related to my test code, I now change it to multiple test programs.
pulsar-broker

bookkeeper-cluster
bookkeeper-cluster-1

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json (hunk 2 above)
Comment (CodePrometheus) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json:217]: I'd like to know more about it, for the overview of clusters, is the method we usually recommend is aggregation display?

Comment (wankai123) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json:217]: Yes, especially the service could have many instances.  If you want to show the all instances overall info, you should
put them on the `Instance List` widget.

Comment (CodePrometheus) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json]: Learned, thanks.

**Review by wankai123 — APPROVED** [review]

**Pull request merged**